06_loops_if_functions.ipynb

```diff
@@ -4,20 +4,21 @@
    "cell_type": "markdown",
```

Comment:
<p>Ya que tenemos impares definido, podemos escribir el codigo como: </p><p> </p><p>for impar in impares:</p><p>print(impar)</p><p> </p><p>Así hacen la conexion con lo de arriba. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>